syntax = "proto3";

package purchasing;

service Purchase {
    rpc AuthorisePurchase (PurchaseRequest) returns (PurchaseResponse) {}
}

message PurchaseRequest {
    string card_details = 1;
    string charge_amount = 2;
}

message PurchaseResponse {
    string message = 1;
}